skin.json

```diff
@@ -64,6 +64,7 @@
   },
   "responsive": {
     "breakpoints": {
+      "audioOnly": {"id": "audioOnly", "name": "oo-xsmall", "multiplier": 0.7},
```

Comment:
The ```name``` refers to the className that gets applied to the player container to style various components responsively. I reused oo-xsmall for the ```name``` so that the ```oo-xsmall``` styles are used.